cmd/restic/cmd_backup.go

```diff
@@ -692,6 +692,10 @@ func runBackup(opts BackupOptions, gopts GlobalOptions, term *termstatus.Termina
 		p.V("start backup on %v", targets)
 	}
 	_, id, err := arch.Snapshot(gopts.ctx, targets, snapshotOpts)
+	if errors.Cause(err) == context.Canceled {
```

Comment:
This just changes the error message from `unable to save snapshot: [...] context canceled` to `canceled`.